deploy.sh

```
@@ -0,0 +1,265 @@
+echo ""
+
+read -p "Remote Path (e.g., /public_html or /home/user/kanaan): " REMOTE_PATH
+read -p "Domain Name (e.g., kanaanerpgaza.espl.ps): " DOMAIN_NAME
+
+# Optional: SSH Key
+read -p "Use SSH Key instead? (y/N): " USE_SSH_KEY
+if [[ $USE_SSH_KEY =~ ^[Yy]$ ]]; then
+    read -p "SSH Key Path (e.g., ~/.ssh/id_rsa): " SSH_KEY
+    SFTP_KEY="-i $SSH_KEY"
+else
+    SFTP_KEY=""
```

Shellcheck (0.11.0)

[warning] SFTP_KEY appears unused. Verify use (or export if used externally).

(SC2034)